### ISSUE TITLE
gammaray targeted process is grayed out

### ISSUE THREAD

**Issue by roverse** [KDAB/GammaRay#107]
I was able to compile gammaray however launching  gammaray with the command line throws this error.
No probe found for ABI qt4.8-MSVC-release-i686

Also using the gammaray launcher the process list is grayed out

**Comment by roverse** [comment]
bump? :/

**Comment by milianw** [comment]
you probably built gammaray with Qt 5 and thus no probe for qt 4 is available - could that be the case? Or how did you compile gammaray, what was the cmake configure output?

**Comment by vkrause** [comment]
The process list should also give you a tooltip saying why it's grayed out, most likely not finding a suitable probe. "gammaray --list-probes" on the command line or the "Self Test" tab in the launcher should tell you which probes you have built. Mismatching Qt version is one possibility, on Windows you also need matching compiler and debug/release mode.

**Comment by roverse** [comment]
I fixed the issue by installing qt4.8.5 BUT I'm unable to attach to the process "Could not establish connection to remote host: connection refused"

**Comment by roverse** [comment]
Bump

**Comment by aylubimov** [comment]
Hello,
I've run into the same situation.
I've built GammaRay 2.2.1 on WIndows 7 with MSVC 12 and Qt 5.2.1. The only probe available is "qt5.2-MSVC-debug-i686".

My application can only run in release mode (for various reasons). GammaRay wouldn't attach to my running app with the same message "No probe found for ABI qt5.2-MSVC-release-i686".

I tried to cheat and created a COPY of the debug probe, named it qt5.2-MSVC-release-i686, tried to attach again. This time there was no "missing probe" message, but the attempt to attached just timed out.

Is there a way to build the release-mode probe? Is there any other workaround?

Thank you very much!
Sincerely,
Alex Lubimov
